sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1393,7 +1393,24 @@ class Analyzer(
 
       case q: LogicalPlan =>
```

Comment:
can we just add a case above it?
```
case f: Filter if ResolveAggregateFunctions.containsAggregate(f.condition) => f

case q: LogicalPlan =>
  ...
```

Reply:
> can we just add a case above it?
> 
> ```
> case f: Filter if ResolveAggregateFunctions.containsAggregate(f.condition) => f
> 
> case q: LogicalPlan =>
>   ...
> ```

Changed, for containsAggregate() method, where I put it is better? or add a common interface for Resolve Reference